lib/ditto.ts

```diff
@@ -33,23 +39,29 @@ const COMMANDS = [
       },
     ],
   },
+  {
+    name: "replace",
+    description: "Find and replace Ditto text with code",
+  },
 ] as const;
 
 const setupCommands = () => {
   program.name("ditto-cli");
 
-  COMMANDS.forEach((config) => {
+  COMMANDS.forEach((command) => {
```

Comment:
I still think this variable fits better as being thought of as a "command configuration" or "command group" since we configure nested commands inside of it categorically, but don't feel super strongly!

Reply:
Makes sense!